torch/fx/passes/infra/pass_manager.py

```diff
@@ -31,6 +31,8 @@ def wrapped_fn(gm):
         fn(gm)
         return PassResult(gm, True)
 
+    if wrapped_fn.__name__ == 'wrapped_fn':
```

Comment:
is this really need?IIUC, functools.wraps doesn't change the wrapped function's name.

Reply:
For the passes that are written through PassBase, they don't have a .__name__ field, so the name will end up being "wrapped_fn". This way they will become the name of the object.